cases/drp-00-05.tex

```diff
@@ -3,16 +3,24 @@ \subsection{DRP-00-05: Execution of the DRP Science Payload by the DRP Execution
 
 \subsubsection{Requirements}
 
-DMS-REQ-0293,DMS-REQ-0294.
+DMS-REQ-0106,DMS-REQ-0292,DMS-REQ-0293,DMS-REQ-0294,DMS-REQ-0302,DMS-REQ-0303,DMS-REQ-0304.
 
 \subsubsection{Test items}
 
-This test will check:
+This test will check that the DRP Science Payload can be executed using the current state-of-the-art execution service provided by the LSST Data Facility.
 
-\begin{itemize}
+Specifically, this will check if the following requirements are fulfilled:
 
-  \item{That the DRP Science Payload can be executed using the current
-  state-of-the-art execution service provided by the LSST Data Facility.}
+\begin{itemize}
+  \item{For each Coadded Image, DMS shall store: the list of input images and the pipeline parameters, including software versions, used to derive it, and a sufficient set of metadata attributes for users to re-create them in whole or in part (DMS-REQ-0106).}
+  \item{To reduce the likelihood for confusion, all IDs shall be unique across databases and database versions, other than those corresponding to uniquely identifiable entities (i.e., IDs of exposures) (DMS-REQ-0292).}
+  \item{A Dataset may consist of one or more pixel images, a set of records in a file or database, or any other grouping of data that are processed or produced as a logical unit. The DMS shall be able to identify and retrieve complete, consistent datasets for processing (DMS-REQ-0293).}
+  \item{The DMS shall process all requested datasets until either a successful result is recorded or a permanent failure is recognized. If any dataset is processed, in part or in whole, more than once, only one of the wholly processed results will be recorded for further processing.
+(DMS-REQ-0294).}
```

Comment:
Actual testing of this requirement was not planned in this milestone.   In order to verify DMS-REQ-0294 in some future test, yes we would have to resubmit the same pipeline multiple times (with and without errors).    I assume since it is a requirement, someone will have to someday do this test.    It sounds like we either need to temporarily remove this requirement from this document or modify the procedure to include the steps.